### ISSUE TITLE
Fee bump transactions 

### ISSUE THREAD

**Issue by MaxHillebrand** [joinmarket-webui/jam#652]
**Is your feature request related to a problem? Please describe.**

Fee bumping is useful 

**Describe the solution you'd like**

Exposé the RBF script:
https://github.com/JoinMarket-Org/joinmarket-clientserver/releases/tag/v0.9.10

**Describe alternatives you've considered**

Not sure if there is CPFP script for incoming transactions.

**Comment by kristapsk** [comment]
> Exposé the RBF script

No, Jam should not do that, JM should add fee bumping support to RPC API.

> Not sure if there is CPFP

Not yet.

**Comment by kristapsk** [comment]
https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/1542
